Makefile

```diff
@@ -117,7 +117,7 @@ lint: lint-contracts-rs
 
 .PHONY: audit
 audit:
-	$(CARGO) audit --ignore RUSTSEC-2021-0073 --ignore RUSTSEC-2021-0076
+	$(CARGO) audit
```

Comment:
💪 🎉

Reply:
Should have been removed when libp2p got removed.  I forgot, and seems like it was never ticketed :man_facepalming: